bms_checkstate.c

```diff
@@ -0,0 +1,103 @@
+#include "bms.h"
+
+/****************************************************************************************
+*Func desc : This function check if the passed battery parameter value is within the specified range 
+*Param     : param_value - The measured battery parameter value, whole limits are to be verified - float type
+*Return    : Returns the status of param limit check
```

Comment:
Nice comments. What do you think of an alternative to have #define / constants to name the 0, 1 and 2?

Reply:
Updated sudeep